CMakeLists.txt

```diff
@@ -345,12 +366,19 @@ IF( NOT BUILD_DOCUMENTATION_ONLY )
     Qt5::Concurrent
     Qt5::Test
     Qt5::Sql
+    Qt5::Network
     ${ITK_LIBRARIES}
     ${VTK_LIBRARIES}
     ${OpenCV_LIBRARIES}
     ${OpenMP_LIBRARIES}
   )
 
+  IF (WIN32) 
+    ADD_LIBRARY(elzip ${11Zip_Headers} ${11Zip_HeadersCpp} ${11Zip_Sources} ${11Zip_SourcesCpp})
+
+    SET( DEPENDENT_LIBS ${DEPENDENT_LIBS} elzip)
+  ENDIF()
+
```

Comment:
1) Do we really need an external lib for this? We might be able to do this with Qt??

2) How are we doing this for non Windows platforms?

It might be much cleaner if you find a Qt based solution.

Reply:
So the Qt solution would be QuaZip. But then this is also an external lib and not Qt built-in, so I have to either put QuaZip in the superbuild, or include the source code in CaPTk like this. I picked 11Zip because it is very easy to use (literally 1 line of code), compare to LibZip and QuaZip.